android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -46,13 +46,16 @@ class TranslationsMiddleware(
         // Pre process actions
         when (action) {
             is InitAction ->
-                scope.launch {
-                    requestEngineSupportAndInit(context)
-                }
+                context.store.dispatch(TranslationsAction.InitTranslationsBrowserState)
 
             is TranslationsAction.InitTranslationsBrowserState -> {
                 scope.launch {
-                    requestSupportedLanguages(context)
+                    val engineIsSupported = requestEngineSupport(context = context)
```

Comment:
nit: I don't think we need the named parameter `context = context` here. Usually that is reserved for when we need to have multi-line argument lists

Reply:
Updated, ty!